### PR TITLE
Added WMR Standalone section to Build and Deploy doc

### PR DESCRIPTION
**Pull request by CDiaz-MS**
## Overview

Added a new section to the Build and Deploy doc that outlines the steps required to use the Windows Mixed Reality Headset for Standalone builds.  Also added Leap Motion to the table of contents.

## Changes
- Fixes: #7685

### CODE REVIEW

**Comment by keveleigh** [comment]
>Azure Pipelines could not run because the pipeline triggers exclude this branch/path.

Updated the branch name so this works now!

File: Documentation/BuildAndDeploy.md [hunk 1]
```diff
@@ -16,3 +16,51 @@ be changed inside the Visual Studio solution).
 
 Make sure that the "Target SDK Version" dropdown includes the option "10.0.18362.0" - if this is missing,
 [the latest Windows SDK](https://developer.microsoft.com/windows/downloads/windows-10-sdk) needs to be installed.
+
+
+## Building and deploying MRTK to a Windows Mixed Reality Headset
+
+The Windows Mixed Reality (WMR) headset can be used for Universal Windows Platform (UWP) and Standalone builds.  A Standalone build for a WMR headset requires the following extra steps:
```
Comment (keveleigh) [inline Documentation/BuildAndDeploy.md:23]: Small note is that XR SDK also supports native WMR in Standalone builds, without needing SteamVR and the WMR plugin. These steps are required for Unity's legacy XR though.
Not sure if we want to mention this somewhere or not.

Comment (CDiaz-MS) [reply Documentation/BuildAndDeploy.md:23]: We can mention it here, I added a note

**Pull request merged**